Documentation/Development.md

```diff
@@ -57,7 +57,7 @@ using a [snapshot](https://swift.org/download/#releases) from swift.org.
     Note: Make sure the directory for llbuild is called "llbuild" and not
     "swift-llbuild".
  
-4. Building the Swift Package Manager.
```

Comment:
Great catch! Let's be consistent when to use the gerund or no

Reply:
I went through and tried to change H2 level headings to be verb phrases, and used the imperative tense for steps in a process.